pkgs/servers/icingaweb2/default.nix

```diff
@@ -23,7 +23,7 @@ stdenvNoCC.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/share
     cp -ra application bin etc library modules public $out
-    cp -ra doc $out/share
+    cp -ra doc schema $out/share
```

Comment:
How about creating a [split package](https://nixos.org/manual/nixpkgs/stable/#chap-multiple-output) instead? Sometimes, you don't want to have the docs or the schema.

Reply:
Yes... and sometimes the PHP application presents you a stacktrace in a dashlet (out of the box) because a hook doesn't find its schema. 🤷‍♂️